components/modal/__examples__/header-footer.jsx

```diff
@@ -23,58 +23,63 @@ const Example = createReactClass({
 	},
 
 	render () {
+		const modal = this.state.noHeaderIsOpen ? (
```

Comment:
`aria-hidden=true` isn't applied when there is more than one modal rendered.  This is actually a bug in react-modal, which is fixed in v2.0.6.  But when I tried updating to it, I saw some test failures (something about onClose handler not being triggered) that I didn't spend too much time looking into.  The right thing to do is to probably fix up our code so that we can use the newer version of react-modal.